mlshvch/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+file = File.open('data.txt', 'r')
+data = file.read
+file.close
+
+EXIT_COMMAND = 'exit!'
+ERROR_MESSAGE = 'City name can not be blank!'
+DEFAULT_VALUE = -1.0
+
+def extract_data(data_string)
```

Comment:
In addition to comment about `@rainfall_data` (see below): you've got options how to do it properly:
* make a function, where you read the file line by line and process it as it gets read;
* read the whole the whole at once and then use this method to extract the contents.

I like the first option more, but the second will do too.